SPECS/swig/swig.spec

```diff
@@ -1,15 +1,14 @@
-Summary:	Connects C/C++/Objective C to some high-level programming languages
-Name:		swig
-Version:	3.0.12
-Release:        4%{?dist}
-License:	GPLv3+
-URL:		http://swig.sourceforge.net/
-Source0:	http://downloads.sourceforge.net/project/swig/swig/swig-%{version}/swig-%{version}.tar.gz
-%define sha1 swig=5cc1af41d041e4cc609580b99bb3dcf720effa25
+Summary:        Connects C/C++/Objective C to some high-level programming languages
+Name:           swig
+Version:        4.0.2
+Release:        1%{?dist}
+License:        GPLv3+
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-BuildRequires:	pcre-devel
-Requires:	pcre
+URL:            http://swig.sourceforge.net/
+Source0:        http://downloads.sourceforge.net/project/swig/swig/swig-%{version}/swig-%{version}.tar.gz
```

Comment:
Please change the URLs to HTTPs, if possible.

Reply:
Fixed